tests/node/index.js

```diff
@@ -0,0 +1,32 @@
+var Fs = require('fs');
+var Path = require('path');
+var Env = require('../..').Env;
+var Fetch = require('../..').fetch;
+
+
+var env = new Env('en-US', Fetch);
+var ctx = env.createContext([
+  Path.join(__dirname + '/fixtures/{locale}/entities.l20n')
+]);
+var langs = [
+  {code: 'es-ES'},
+  {code: 'en-US'},
+  {code: 'fr'}
+];
+
+ctx.fetch(langs).then(init);
+
+function init () {
+  ctx.resolve(langs, 'whatCanIAsk').then(function (value) {
+    console.log('SINGLE VALUE: ', value);
+  });
```

Comment:
The reason why we currently require `langs` as the first argument are race conditions. It allows us to not have to keep and synchronize `langs` across multiple contexts.  But I'm open to discussion if you have better ideas.